Sources/Testing/Traits/Tags/Tag.Color+Loading.swift

```diff
@@ -49,22 +49,28 @@ private var _appDataDirectoryPath: String? {
 /// On Apple platforms and on Linux, this path is equivalent to
 /// `"~/.swift-testing"`. On Windows, it is equivalent to
 /// `"%HOMEPATH%\AppData\Local\.swift-testing"`.
-var swiftTestingDirectoryPath: String {
+/// Returns `nil` if the platform does not support the concept of a user
```

Comment:
```suggestion
/// The value of this property is `nil` if the platform does not support the concept of a home directory, or if the home directory could not be determined.
```

(Wrap to 80 columns.)